test/unit/modules/system.js

```diff
@@ -26,12 +26,23 @@ describe('system', () => {
 	let __private;
 	let self;
 	let loggerStub;
+	let childLoggerStub;
 	let dbStub;
 	let dummyConfig;
 
 	beforeEach(done => {
 		// Library
+		childLoggerStub = {
+			done: sinonSandbox.spy(),
```

Comment:
what is `done` log level? I saw this at 3 different places above as well.

Reply:
I guess forgotten from the last library